bool positionA
bool positionB
bool positionC
